service/routes.py

```diff
@@ -199,34 +199,24 @@ def delete_products_from_wishlist(wishlist_id):
 ######################################################################
 # Delete a product from a wishlist
 ######################################################################
-@app.route("/wishlists/<int:wishlist_id>/products/<int:product_id>", methods=["DELETE"])
-def delete_a_product_from_wishlist(wishlist_id, product_id):
```

Comment:
This should remain the same, otherwise the URL is not restful. We can internally look up products by ID, but this should stay as it was. I figured out a way to copy both things in my pull request, please take a look.